modules/admin_manual/pages/configuration/files/encryption/encryption_configuration_quick_guide.adoc

```diff
@@ -53,17 +53,18 @@ Depending on the amount of existing data, this operation can take a long time.
 {occ-command-example-prefix} app:disable encryption
 ----
 
-If the master key has been compromised or exposed, you can recreate it.
-You will need the current master key for it.
+If the master key has been compromised or exposed, you can recreate it. You will need the current master key for it.
```

Comment:
Recreation when exposed does not make much sense to me. I'd assume:
* One would want to "recreate it" when it was lost.
* But when it was compromised, one would rather create a new version of the master key?

Reply:
I would write the diplomatic version and agree to @EParzefall using "replace" - any obstacles?

Reply:
Made a committable PR and used replace. Any improvements can be done later too.